pkgs/servers/nextcloud/packages/25.json

```diff
@@ -10,29 +10,29 @@
     ]
   },
   "calendar": {
-    "sha256": "0xhrpadzz73rdjyk4y1xm5hwc6k104rlpp9nmw08pq8phpfs12qa",
-    "url": "https://github.com/nextcloud-releases/calendar/releases/download/v4.3.3/calendar-v4.3.3.tar.gz",
-    "version": "4.3.3",
+    "sha256": "154xw925v8wipl472vpksiccf1jnic615add6v1zzaxwqwclhjrv",
+    "url": "https://github.com/nextcloud-releases/calendar/releases/download/v4.4.0-beta1/calendar-v4.4.0-beta1.tar.gz",
+    "version": "4.4.0-beta.1",
```

Comment:
Is there a good reasaon to upgrade to betas?

Reply:
the script just updates to the latest available version regardless of it is a beta

Reply:
Tracked the issue here https://nixos.wiki/wiki/Nextcloud#Known_issues , upstream https://github.com/helsinki-systems/nc4nix/issues/4 and trying to create a PR for it to fix it :)

Reply:
They use GitLab, this is a readonly mirror